.github/workflows/linux.yml

```diff
@@ -182,7 +182,103 @@ jobs:
           path: ${{ env.BUILD_DIR }}/openvino_package.tar.gz
           if-no-files-found: 'error'
 
-  openvino_tokenizers:
+  openvino_tokenizers_cpack:
+    name: OpenVINO tokenizers cpack (BUILD_FAST_TOKENIZERS=${{ matrix.build_fast_tokenizers }}, BUILD_TYPE=${{ matrix.build_type }})
+    strategy:
+      matrix:
+        build_fast_tokenizers: [ON]
```

Comment:
why do we need to build fast tokenizers?
We test build inside wheel stage already

Reply:
To force tokenizers build, in wheel package it is enabled for Linux only (based on centos package)